### PR TITLE
Range and qos

### PR DESCRIPTION
**Pull request by fmrico**
**Description**
This PR contains:
* support for RangeFinder
* Fix camera_info content and publication rate
* Fix QoS

**Affected Packages**
List of affected packages:
  - webots_ros2_core

### CODE REVIEW

**Review by lukicdarkoo — CHANGES_REQUESTED** [review]
Thank you very much for this nice contribution!

I did a few minor fixes to make the CI pass.

Before the merge, can you tell me why did you change `wheel_radius` in `tiago.yaml`?

File: webots_ros2_tiago/resource/tiago.yaml [hunk 1]
```diff
@@ -8,4 +8,4 @@ webots_driver:
     synchronization: false
     use_joint_state_publisher: true
     wheel_distance: 0.404
-    wheel_radius: 0.1955
+    wheel_radius: 1
```
[the remark this answers is not in the document]
Comment (fmrico) [reply webots_ros2_tiago/resource/tiago.yaml:11]: Hi Lukic, I will push now the values I used.

**Comment by fmrico** [comment]
Hi @lukicdarkoo 

I modified the `wheel_distance` and `wheel_radius` to approximate reasonable values because the original values made the odometry updates fail. I got to map a complex environment with these values.

Best

**Comment by lukicdarkoo** [comment]
> Hi @lukicdarkoo
> 
> I modified the `wheel_distance` and `wheel_radius` to approximate reasonable values because the original values produced erroneous updates of the `odom`->`base_link` transform. I got to map a complex environment with these values.
> 
> Best

In that case, the problem is in `webots_differential_drive_node.py`. I will check what is happening. It works fine for TurtleBot and e-puck.

**Review by lukicdarkoo — APPROVED** [review]
Looks good, thank you. Feel free to merge it.

P.S. I see that the odometry parameters slightly deviated from the original values. Good to know that these values work better.

**Comment by fmrico** [comment]
Hi @lukicdarkoo 

I think that I have no permissions to merge it. Could you merge it for me, please?

Thanks for your help ;)

**Comment by omichel** [comment]
@fmrico: I just added you in the committers group, so that you should be able to accept my invitation and merge this PR (and others).

**Comment by fmrico** [comment]
Thanks, @omichel  ;)

**Pull request merged**